1. WAP to print the “hello world” for 5 times by using a while loop.

In [ ]:
start = 1
end = 5
while start<=end:
    print("Hello World")
    start += 1

2. WAP to print the number from 1 to 10 by using a while loop.

In [ ]:
start = 1 
end = 10 
while start<=end:
    print(start)
    start+=1

3. WAP to print the natural number until 1 to nth times by using a while loop.

In [ ]:
start = 1 
end = eval(input())
while start<=end:
    print(start)
    start += 1


4. WAP to print the reverse of the natural number nth to 0 times by using a while loop.

In [ ]:
start = eval(input())
end = 0 
while start>=end:
    print(start)
    start-=1 

5. WAP to print the whole number until nth times by using a while loop.

In [ ]:
start = 0 
end = eval(input())
while start<=end:
    print(start)
    start+=1

6. WAP to print the reverse of the whole number until nth times by using a while loop.

In [ ]:
start = eval(input())
end = 0 
while start>=0:
    print(start)
    start-=1

7. WAP to print the even number until nth times by using a while loop.


In [ ]:
end = eval(input())
start = 2 
while start<=end:
    if start%2==0:
        print(start)
    start+=1 
    


8. WAP to print reserve the even number until nth times by using a while

In [ ]:
start = eval(input())
end = 2 
while start>=end:
    if(start%2)==0:
        print(start)
    start-=1

9. WAP to print the odd number until nth times by using a while loop.

In [ ]:
start = 1 
end = eval(input())
while start<=end:
    if(start)%2!=0:
        print(start)
    start+=1

10. WAP to print reserve the odd number until nth times by using a while loop.

In [ ]:
start = eval(input())
end = 1 
while start>=end:
    if(start%2)!=0:
        print(start)
    start-=1
        

11. WAP to print the nth table 1 to 20, format ==> 5 * 1 = 5 to 5 * 20 = 100

In [54]:
n = eval(input())
start = 1 
while start<=20:
    print(str(n) + " X " +str(start)+ " = " + str(n*start))
    start+=1

7 X 1 = 7
7 X 2 = 14
7 X 3 = 21
7 X 4 = 28
7 X 5 = 35
7 X 6 = 42
7 X 7 = 49
7 X 8 = 56
7 X 9 = 63
7 X 10 = 70
7 X 11 = 77
7 X 12 = 84
7 X 13 = 91
7 X 14 = 98
7 X 15 = 105
7 X 16 = 112
7 X 17 = 119
7 X 18 = 126
7 X 19 = 133
7 X 20 = 140


12. WAP to print the nth table 20 to 1, format ==> 5 * 20 = 100 to 5 * 1 = 5

In [55]:
n = eval(input())
start = 20 
end = 1 
while start>=end:
    print(str(n)+ "*" + str(start) + "=" + str(n*start))
    start -=1
    

9*20=180
9*19=171
9*18=162
9*17=153
9*16=144
9*15=135
9*14=126
9*13=117
9*12=108
9*11=99
9*10=90
9*9=81
9*8=72
9*7=63
9*6=54
9*5=45
9*4=36
9*3=27
9*2=18
9*1=9


13. WAP to print the numbers which are divisible by 3 from 25 to 66.

In [ ]:
start = 25 
end = 66 
while start<=end:
    if start%3 ==0:
        print(start,end=" ")
    start += 1 

14. WAP to print the numbers which are divisible by 3 and 5 from 15 to 76.

In [ ]:
start = 15 
end = 76 
while start<=end:
    if start%3 ==0 and start%5==0:
        print(start,end=" ")
    start += 1 

15. WAP to print the numbers which are divisible by 5 and 7 from 77 to 22.

In [ ]:
start = 77 
end = 22 
while start>=end:
    if start%3 ==0 and start%7==0:
        print(start,end=" ")
    start -= 1 

16. WAP to print the ”love you” if value is divisible by 3, or print “hate you” value is
divisible by 5, to print “i like you” if divisible by 3 and 5, from 1 to 30.

In [ ]:
start = 1 
end = 30 
while start<=end:
    if start%3==0:
        print("I love you")
    elif start%5==0:
        print("I hate you")
    elif start%3==0 and start%5==0:
        print("I like you")
    start+=1

In [56]:
start = 1 
end = 30 
while start<=end:
    if start%3==0:
        print(start)
    elif start%5==0:
        print(start)
    elif start%3==0 and start%5==0:
        print(start)
    start+=1

3
5
6
9
10
12
15
18
20
21
24
25
27
30


17. WAP to print sum of the 1 to nth natural numbers.

In [ ]:
start = 1 
end = eval(input())
sum = 0 
while start<=end:
    sum = sum + start 
    start+=1 
print(sum)
    

18. WAP to print sum of the even numbers 1 to 20 numbers.

In [ ]:
start = 2 
end = eval(input())
sum = 0
while start<=end:
    if(start%2)==0:
        sum = sum + start
    start+=1 
print(sum)
    

19. WAP to print products of the 1 to 6 numbers.

In [ ]:
start = 1 
end = 6
prod = 1 
while start<=end:
    prod = prod * start
    start += 1
print(prod)     


20. WAP to count the no of odd values from 1 to 30 numbers.

In [ ]:
start = 2 
end = 30 
count = 0 
while start<=end:
    if(start)%2!=0:
        count+=1 
    start+=1 
print(count)

21. WAP to print the factors of the given number.

In [57]:
a = eval(input())
start = 1 
while start<=a:
    if(a%start)==0:
        print(start)
    start += 1 

1
2
3
6


22. WAP to sum of the factors of the given number.

In [ ]:
a = eval(input())
start = 1 
sum = 0 
while start<=a:
    if(a%start)==0:
        sum = sum + start
    start+=1
print(sum)

23. WAP to prod of the factors of the given number.

In [ ]:
a  = eval(input())
start = 1 
prod = 1 
while start<=a:
    if(a%start)==0:
        prod = prod * start
    start+=1 
print(prod)

24. WAP to print Fibonacci series 0 to nth.

In [58]:
end = eval(input())
while end<=0:
    if end < 0:
        print("Invaild number")
    elif end == 0 :
        print(0)
    elif  end == 1 or end == 2 :
        print(1)
    else:
        print((end-1)+(end-2))

25. WAP to find out the factorial number of a given number.

In [59]:
number = eval(input())
start = 1 
prod = 1 
while start<=number:
    prod = prod * start
    start += 1
print(prod)

5040


26. WAP to print the factorial numbers in the given range m to n.

In [60]:
m = eval(input())
n = eval(input())
prod = 1 
while m<=n:
    prod = prod * m 
    m+=1 
print(prod)

1


27. WAP to check whether a given number is a prime number or not.

In [ ]:
a = eval(input())
factors = 0 
start = 1 
while start<=a:
    if(a%start)==0:
        factors = factors + 1 
    start+=1 
if(factors==2):
    print("Prime number")
else:
    print("Not a prime number")

28. WAP to print the prime numbers in the given range m to n.

In [ ]:
m = eval(input())
n  = eval(input())
while m<=n:
    i = m
    start = 2 
    while start<i:
        if(i%start)==0:
            break
        start += 1 
    else:
        print(m,end = " ")
    m+=1

29. WAP to check whether a given number is an armstrong number or not.

In [ ]:
n = int(input())
temp = n
temp1 = n 
count = 0 
while n:
    n = n //10
    count += 1 
sum1 = 0 
while temp:
    d = temp%10
    sum1 = sum1 + (d**count)
    temp = temp//10 
if sum1 == temp1:
    print("Armstong number")
else:
    print("Not a Arm strongnumber")


30. WAP to print the armstrong numbers in the given range m to n.

In [7]:
i = 10
j = 2000
while i<=j:
    n = i
    temp = i
    count = 0 
    while n:
        n  = n//10
        count+=1
    sum = 0 
    while temp:
        d = temp%10
        sum = sum + (d**count)
        temp = temp//10 
    if sum == i:
        print(sum)
    i+=1

153
370
371
407
1634


31. Write a program to print prime series 1 to nth.

In [5]:
num1 = 2
num2 = 1000
while num1<=num2:
    m = num1 
    start = 2 
    while start<m:
        if(m%start)==0:
            break 
        start += 1
    else:
        print(num1,end = " ")
    num1+=1

2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71 73 79 83 89 97 101 103 107 109 113 127 131 137 139 149 151 157 163 167 173 179 181 191 193 197 199 211 223 227 229 233 239 241 251 257 263 269 271 277 281 283 293 307 311 313 317 331 337 347 349 353 359 367 373 379 383 389 397 401 409 419 421 431 433 439 443 449 457 461 463 467 479 487 491 499 503 509 521 523 541 547 557 563 569 571 577 587 593 599 601 607 613 617 619 631 641 643 647 653 659 661 673 677 683 691 701 709 719 727 733 739 743 751 757 761 769 773 787 797 809 811 821 823 827 829 839 853 857 859 863 877 881 883 887 907 911 919 929 937 941 947 953 967 971 977 983 991 997 

32. Write a program to print 1 to 100 prime numbers in fibonacci series

In [7]:
a = 0 
b = 1
print(a,b,end=" ") 
start = 2 
end = 100 
while start<=end:
    n = start
    j = 2 
    while j<n:
        if(n%j)==0:
            break 
        j+=1
    else:
        c = a + b
        a = b 
        b = c 
        print(c,end=" ")
    start+=1 
# while start<=n:
#     c = a + b
#     a = b 
#     b = c 
#     print(c,end=" ")
#     start+=1 

0 1 1 2 3 5 8 13 21 34 55 89 144 233 377 610 987 1597 2584 4181 6765 10946 17711 28657 46368 75025 121393 

33. Write a program Sum of prime numbers in a given collection

In [12]:
coll = [2,3,5,7,11,13]
start = 0 
end = len(coll)
sum = 0
for i in range(start,end):
    sum = sum + coll[i] 
print(sum)
    

41


34. WAP to the given number is palindrome or not.

In [15]:
n = 121
temp = n  
res = 0 
while n!=0:
    d = n%10
    res = d + res * 10 
    n = n //10 
if(temp==res):
    print("palindrome number")
else:
    print("Not a palindrome number")
    


palindrome number


35.WAP to check whether the length of the string is even or odd. If it is even, print
that string 10 times. If it is not printed 5 times.

In [20]:
word = "vamshikrishna"
end = len(word)
if end%2==0:
    word = word + " "
    print(word*10)
else:
    word = word + " "
    print(word*5)

vamshikrishna vamshikrishna vamshikrishna vamshikrishna vamshikrishna 


36.WAP to print the uppercase character from ‘A’ to ‘Z’ by using a while loop.

In [24]:
start = 'A'
end = 'Z'
while start<=end:
    print(start,end=" ")
    start = chr(ord(start)+1)
    

A B C D E F G H I J K L M N O P Q R S T U V W X Y Z 

37. WAP to print the lowercase character from ‘a’ to ‘z’ by using a while loop.

In [26]:
start = 'a'
end = 'z'
while start<=end:
    print(start,end=" ")
    start = chr(ord(start)+1)

a b c d e f g h i j k l m n o p q r s t u v w x y z 

In [35]:
start = 'a'
end = 'z'
for i in range(65,91):
    print(chr(i),end =" ")
    

A B C D E F G H I J K L M N O P Q R S T U V W X Y Z 

38. WAP to print the ascii numbers character from ‘0’ to ‘9’ by using a while loop.

In [36]:
start = '0'
end = '9'
while start<=end:
    print(start,end = " ")
    start = chr(ord(start)+1)

0 1 2 3 4 5 6 7 8 9 

39. WAP to print the uppercase character from ‘E’ to ‘U’ by using a while loop.

In [37]:
start = 'E'
end = 'U'
while start<=end:
    print(start,end = " ")
    start = chr(ord(start)+1)

E F G H I J K L M N O P Q R S T U 

40. WAP to print the uppercase character from ‘Z’ to ‘A’ by using a while loop.

In [46]:
start = 'Z'
end = 'A'
while start>=end: 
    print(start,end=" ")
    start = chr(ord(start)-1)


ZYXWVUTSRQPONMLKJIHGFEDCBA

41. WAP to print the uppercase character from ‘U’ to ‘G’ by using a while loop.

In [3]:
start = 'U'
end = 'G' 
while start>=end:
    print(start,end=" ")
    start = chr(ord(start)-1)

U T S R Q P O N M L K J I H G 

42. WAP to print the lowercase character from ‘i’ to ‘y’ by using a while loop.

In [6]:
start = 'i'
end = 'y'
while start<=end:
    print(start,end = " ")
    start = chr(ord(start)+1)

i j k l m n o p q r s t u v w x y 

43. WAP to print the lowercase character from ‘z’ to ‘a’ by using a while loop.

In [7]:
start = 'z'
end = 'a'
while start>=end:
    print(start,end =" ")
    start = chr(ord(start)-1)

z y x w v u t s r q p o n m l k j i h g f e d c b a 

44. WAP to print the lowercase character from ‘p’ to ‘c’ by using a while loop.

In [8]:
start = 'p'
end = 'c'
while start>=end:
    print(start,end = " ")
    start = chr(ord(start)-1) 

p o n m l k j i h g f e d c 

45. WAP to print the uppercase characters based on even ascii values.

In [24]:
start = 65
end = 91
while start<=end:
    if(start)%2==0:
        print(chr(start),end= " ")
    start += 1

B D F H J L N P R T V X Z 

46. WAP to print the lowercase characters based on even ascii values.

In [25]:
start = 98 
end = 122
while start<=end:
    if(start)%2==0:
        print(chr(start),end = " ")
    start += 1

b d f h j l n p r t v x z 

47. WAP to print the uppercase characters based on odd ascii values.

In [30]:
start = 65 
end = 91 
while start<=end:
    if(start)%2!=0:
        print(chr(start),end=" ")
    start+=1

A C E G I K M O Q S U W Y [ 

48. WAP to print the lowercase characters based on odd ascii values.

In [31]:
start = 97 
end = 122
while start<=end:
    if(start%2)!=0:
        print(chr(start),end = " ")
    start += 1 

a c e g i k m o q s u w y 

49. WAP to print all the even characters from ‘A’ to ‘Z’.

In [48]:
start = ord('A')
end = ord('Z')
while start<=end:
    if(start)%2==0:
        print(chr(start),end=" ")
    start +=1

B D F H J L N P R T V X Z 


50. WAP to print all the odd characters from ‘a’ to ‘z’.

In [50]:
start = ord('a')
end = ord('z')
while start<=end:
    if(start)%2 == 0:
        print(chr(start),end = " ")
    start+=1 

b d f h j l n p r t v x z 






57. Write a java program to find the largest number ‘M’ less than a given number ‘N’
which should not contain a given digit ‘D’.
For example, if 150 is the given number and 5 is the given digit, then you should
find the largest number less than 150 such that it should not contain 5 in it. In this
case, 149 will be the answer.


60. WAP to print the uppercase characters in a given string.

51. WAP to count the number of vowels present in D to V.

In [1]:
start = 'D'
end = 'V'
count = 0 
while start<=end:
    if(start in 'aeiouAEIOU'):
        count += 1
        print(ord(chr(count)))
    start = chr(ord(start)+1) 
    

1
2
3
4


In [68]:
start = 'p'
end = 'c'
while start>=end:
    print(start,end = " ")
    start = chr(ord(start)-1) 

p o n m l k j i h g f e d c 

In [72]:
v = 'v'
if v == 'vamshi':
    print("True")

In [79]:
r = 65
print(chr(r))

A


52. WAP to count the number of vowels present in u to e.

In [17]:
start = 'u'
end = 'e'
count = 0
while start>=end:
    if start in 'aeiouAEIOU':
        count +=1 
        print(count,end=" ")
    start = chr(ord(start)-1)
        #print(start,end = " ")

1 2 3 4 

53. WAP to print the square of the number in between the range from m to n which
is multiple of 3.


In [19]:
m = int(input())
n = int(input())
while m<=n:
    if(m%3)==0:
      print(m**3)
    m+=1

216


 
54. Write a program which takes an integer number and find the sum of digits and
repeat until the sum gets a single digit in the end.
Example:
Input 5643
Example:
5+6+4+3 =>18
1+8=>9
Output 9

In [38]:
a = 5643
while a>9:
    digit_sum = 0 
    while a > 0:
        digit_sum += a % 10
        a = a//10
    a = digit_sum
print(a)

9


In [37]:
num = 5643

while num > 9:
    digit_sum = 0
    while num > 0:
        digit_sum += num % 10
        num //= 10
    num = digit_sum

print("Output:", num)


Output: 9


55. Write a program to count the numbers of 2’s between 0 and n
Input n=35
Example: 2,12,20,21,22,23,24,25,26,27,28,29,32
Output 14

In [42]:
n = 35
count = 0 
start = 2
while start<=n:
    i = start 
    while i > 0:
        if i %10 == 2:
            count +=1 
        i = i //10
    start += 1
print(count)
    

14


In [43]:
n = 35 
start = 2 
count = 0
while start <= n:
    i = start
    while i >0:
        if(i%10)==2:
            count +=1 
        i = i //10
    start += 1 
print(count)   

14


56. Write a program to find max product of three numbers from the given integer
array
Input arr() =6,3,2,0,10
Example: 10*6*2
Output =180

In [52]:
list_1 = [6,3,2,0,10]
list_1.sort()
mult = list_1[-1] * list_1[-2] * list_1[-3]  
print(mult)

180


57. Write a python program to find the largest number ‘M’ less than a given number ‘N’
which should not contain a given digit ‘D’.
For example, if 150 is the given number and 5 is the given digit, then you should
find the largest number less than 150 such that it should not contain 5 in it. In this
case, 149 will be the answer.

In [58]:
N = 150
D = 5

M = N - 1

while str(D) in str(M):
    M -= 1

print("Output:", M)


Output: 149


58. WAP to print all the characters in a given string.

In [68]:
a = 'Vamshi'
start = 0
while start<len(a):
    print(a[start])
    start+=1

V
a
m
s
h
i


In [69]:
n = 6  # Number of rows in the pattern

for i in range(1, n + 1):
    for j in range(1, n + 1):
        if i == j or i + j == n + 1:
            print("*", end=" ")
        else:
            print(" ", end=" ")
    print()


*         * 
  *     *   
    * *     
    * *     
  *     *   
*         * 


In [72]:
n = 6  # Number of rows in the pattern

for i in range(1, n + 1):
    for j in range(1, n + 1):
        if i == j or i + j == n + 1 or i == 1 or i == n or j == 1 or j == n:
            print("*", end=" ")
        else:
            print(" ", end=" ")
    print()


* * * * * * 
* *     * * 
*   * *   * 
*   * *   * 
* *     * * 
* * * * * * 


In [73]:
rows = 3
columns = 4

for i in range(rows):
    for j in range(columns):
        print(i + 3, end=" ")
    print()


3 3 3 3 
4 4 4 4 
5 5 5 5 


59. WAP to print all the items present in the string between the range from m to n.

In [3]:
class Employee():
    name = "Vamshi"
    marks = 98 
    center = "Hyderabad"
Vamshi = Employee()
print(Vamshi.name)
print(Vamshi.marks)
print(Vamshi.center)
print(id(Employee))
print(id(Vamshi))

Vamshi
98
Hyderabad
1995933068432
1995953727888


In [ ]:
class Employee():
    center = "Not known"
    def __init__(self,name,marks,center):
        self.name = name 
        self.marks = marks 
        self.center = center 
        

In [4]:
class parent1:
    a = 4 
class parent2:
    b = 6 
class child(parent1,parent2):
    c = 9 
ch = child 
print(ch.a)
print(ch.b)
print(ch.c)

4
6
9


In [5]:
class parent:
    a = 12 
class child1(parent):
    b = 13 
class child2(child1):
    c = 14 
ch = child2
print(ch.a)
print(ch.b)
print(ch.c)
    

12
13
14


In [7]:
class employee:
    def __init__(self,a,name):
        self.a = a 
        self.name = name 
    def __add__(self,obj):
        return self.a+obj 
    def __str__(self):
        return self.name 
    def __len__(self):
        return self.a 
a = employee(21,"Krishna")
b = employee(22,"Vamshi")
print(a,b)
print(len(a))
print(len(b))

Krishna Vamshi
21
22


In [1]:
list_1 = [1,2,[5,6,7,8],9,10]
print(list_1[2][2])

7


In [9]:
m = 3
n = 8 
for i in range(m,n+1):
    factors1 = 0 
    for j in range(2,i):
        if(i%j)==0:
            factors1 = factors1 + 1
    if(factors1==0):
        print(i)

3
5
7
